chinese-text-search/README_CN.md

```diff
@@ -0,0 +1,111 @@
+# 基于Transformer的中文文字搜索
+
+<table>
+  <tr>
+    <td>
+      输入
+    </td>
+    <td>
+      一个text文本文件
+    </td>
+  </tr>
+  <tr>
+    <td>
+      输出
+    </td>
+    <td>
+      <pre>top_k</pre> 个匹配查询的句子
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Jina 版本
+    </td>
+    <td>
+      1.0.0
+    </td>
+  </tr>
+</table>
```

Comment:
Give it a try to native markdown table. Currently, it looks wired.
![image](https://user-images.githubusercontent.com/4329072/109732475-34a0ee80-7bf8-11eb-9746-924ec1c41257.png)